src/server/dflycmd.cc

```diff
@@ -120,6 +120,12 @@ void DflyCmd::ReplicaInfo::Cancel() {
     if (flow->cleanup) {
       flow->cleanup();
     }
+    // flow->cleanup() might be set when we transition to StableSync but the saver
```

Comment:
I dont understand this comment
and its not related to big value serialization right?

Reply:
not really but we need to clean/reset the saver. Otherwise the destructor of `TheadLocalMutex` will complain